Changelog.md

```diff
@@ -91,6 +91,9 @@ Compiler Features:
  * Code Generator: Allocate and free local variables according to their scope.
  * Removed ``pragma experimental "v0.5.0";``.
 
+Critical Bugfixes:
```

Comment:
If it's graded as `medium/high` might be better to use the wording "Important Bugfixes" instead of Critical